api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -60,7 +60,10 @@ public UserController(UserService userService) {
         .ofNullable(Sort.Direction.fromStringOrNull(sortOrder))
         .orElse(Sort.Direction.ASC);
     Sort sort = new Sort(new Sort.Order(direction, DEFAULT_SORT_FIELD));
-    List<User> users = userService.findUsersBySearchString(term, sort);
+    List<User> users = userService.findUsersBySearchString(term, sort)
+        .stream()
+        .filter(user -> user.getFirstSignInTime() != null)
```

Comment:
Please add a comment, it is not obvious that we'd be using first sign in time as a proxy for Firecloud user creation.